assets/src/edit-story/components/panels/document/slug/slug.js

```diff
@@ -68,9 +88,10 @@ function SlugPanel() {
     [updateStory]
   );
 
-  const handleChange = useCallback((value) => updateSlug(value, true), [
-    updateSlug,
-  ]);
+  const handleChange = useCallback(
+    (evt) => setSlug(cleanForSlug(evt.target.value, true)),
+    []
+  );
```

Comment:
I currently have this only updating the saved slug when we blur. Do we want to save this every single key stroke?